glossary.asciidoc

```diff
@@ -92,63 +92,61 @@ Breach Remedy Transaction::
     A high capacity does not guarantee that the channel can be used for routing in both directions.
     The capacity does not change if some of the capacity is locked up in HTLCs.
     This means that even a high capacity channel could at any time be unable to route a payment of a certain amount in both directions.
-    It is not to be confused with the balance.
+    Not to be confused with the balance.
 
 c-lightning::
     Implementation of the Lightning Network Protocol by the Victoria based company https://blockstream.com[Blockstream]. It is written in C. Source code is at https://github.com/ElementsProject/lightning.
 
 closing transaction::
     If both channel partners agree to close a channel they will create an exercise settlement transaction that reflects the most recent commitment transaction.
     It does not include any Hashed Time Lock Contracts or Revocable Sequence Maturity Contracts.
-    After exchanging signatures for a closing transaction no further channel updates should be made, as this one allows one side to enforce the closing transaction on the blockchain.
-    Mutually closing a channel with the help of a closing transaction has the advantage that fewer blockchain transactions are required to claim all funds, in comparison to unilaterally forcing a channel close by publishing a commitment transaction. Additionally, funds are for both parties immediately spendable from a closing transaction.
+    After exchanging signatures for a closing transaction, no further channel updates should be made as this one allows one side to enforce the closing transaction on the blockchain.
+    Mutually closing a channel with the help of a closing transaction has the advantage that fewer blockchain transactions are required to claim all funds, in comparison to unilaterally forcing a channel close by publishing a commitment transaction. Additionally, funds for both parties are immediately spendable from a closing transaction.
 
 coinbase::
-    A special field used as the sole input for coinbase transactions. The coinbase allows claiming the block reward and provides up to 100 bytes for arbitrary data.
+    A special field used as the sole input for coinbase transactions. The coinbase allows the block reward to be claimed and provides up to 100 bytes for arbitrary data.
     The block reward consists of two things.
     First, newly generated coins. The amount of allowed coins to be generated is part of the consensus rules and decreases over time based on the current block height.
     Second, the miner is also allowed to add all the fees of the transactions from the current block to the coinbase.
     Not to be confused with coinbase transaction.
 
 coinbase transaction::
-    The first transaction in a block. Always created by a miner, it includes a single coinbase.
+    The first transaction in a block which is always created by a miner and which includes a single coinbase.
```

Comment:
I'm not sure if the modified version is better than the original here.

Reply:
Agreed.  The style of the original came across a quite abript.  The meaning is the same.